web/src/engine/websites/MangaSail_e2e.ts

```diff
@@ -0,0 +1,27 @@
+import { TestFixture, type Config } from '../../../test/WebsitesFixture';
+
+const config: Config = {
+    plugin: {
+        id: 'mangasail',
+        title: 'MangaSail',
+    },
+    container: {
+        url: 'https://www.mangasail.net/content/seoul-station-necromancer',
+        id: '/content/seoul-station-necromancer',
+        title: 'Seoul Station Necromancer'
+    },
+    child: {
+        id: '/content/seoul-station-necromancer-82?page=all',
+        title: '82',
+        timeout: 20_000,
```

Comment:
This hight timeout seems suspicious, isn't there a chapter that loads faster to decrease the test duration.

Reply:
Tried with chapter 83 it took only 11 seconds. Should we settle for 15?

Reply:
Changing manga make it worse, i get timeouted on getting manga step. I dont think my internet has problems tho.